pkg/tracker/tracker.go

```diff
@@ -38,11 +39,13 @@ type FairnessTracker struct {
 func NewFairnessTrackerWithClockAndTicker(trackerConfig *config.FairnessTrackerConfig, clock utils.IClock, ticker utils.ITicker) (*FairnessTracker, error) {
 	st1, err := data.NewStructureWithClock(trackerConfig, 1, trackerConfig.IncludeStats, clock)
 	if err != nil {
+		logger.Printf("error in creating first tracker : %s", err)
 		return nil, NewFairnessTrackerError(err, "Failed to create a structure")
 	}
 
 	st2, err := data.NewStructureWithClock(trackerConfig, 2, trackerConfig.IncludeStats, clock)
 	if err != nil {
+		logger.Printf("error in creating second tracker : %s", err)
```

Comment:
Error being an object, we should use %v to log.
Also, I believe this will not be on a separate line from the next log?